doc/cephfs/mount-using-kernel-driver.rst

```diff
@@ -6,6 +6,12 @@ The CephFS kernel driver is part of the Linux kernel. It allows mounting
 CephFS as a regular file system with native kernel performance. It is the
 client of choice for most use-cases.
 
+.. note:: CephFS mount device string now uses a new (v2) syntax. The mount
+          helper (and the kernel) is backward compatible with the old syntax.
+          This means that the old syntax can still be used for mounting with
+          newer mount helpers and kernel. However, it is recommended to use
+          the new syntax whenever possible.
+
```

Comment:
nit: it may be good to have a section in this file about the legacy syntax. We'll be dealing with people using that syntax for years, so it'd still be good to have a (short) section documenting it, and maybe also explain that when it's used that the transport will default to msgrv1.

Reply:
Makes sense. ACK.

BTW, Using the old syntax as `:/` will result is the mount helper choosing v1 addr from ceph conf. If the user supplies a mon addr (`addr:/`), its used as is it. The new syntax however tries to fetch v2 addrs if no mon addrs are given via mount option and switches to v1 addr if it cannot find any.